### PR TITLE
Updated MDK tutorial

### PR DESCRIPTION
**Pull request by jitendrakansal**
Updated MDK tutorial

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-rest-api/cp-mobile-dev-kit-rest-api.md [hunk 1]
```diff
@@ -87,44 +91,73 @@ With above details, you will next configure an app in Mobile Services, add root
...
+
+    !![MDK](img_0.0.png)
+
+2. Choose `MDKTutorial` as the name for your dev space and **SAP Cloud Platform Mobile Services** as the application type. Continue with **Create Dev Space**.
 
-1. Launch the SAP Web IDE and select the **MDK perspective** by clicking on the icon in the left panel.
+    !![MDK](img_0.1.png)
 
-2. Right-click Workspace folder and select **New** | **MDK Empty Project**.
+3. When it is ready, open your dev space by clicking on the name.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-rest-api/cp-mobile-dev-kit-rest-api.md:118]: We don't say "click on".

```suggestion
3. When it is ready, open your dev space by clicking its name.
```

File: tutorials/cp-mobile-dev-kit-rest-api/cp-mobile-dev-kit-rest-api.md [hunk 2]
```diff
@@ -469,42 +624,90 @@ With above details, you will now create a new MDK rule to create a new Pet recor
...
 [OPTION BEGIN [iOS]]
 
 1. Re-launch the app on your device, authenticate with passcode or Touch ID if asked. You will see a _Confirmation_ pop-up, tap **OK**.
 
-2. Tap **+** icon on `Main.page` to create a new Pet record. A new record has been created and results are displaying in alert pop-up.
+2. Tap **+** icon on `Main.page` to navigate to Create Pet page.
+
+    !![MDK](img_9.1.1.png)
+
+3.  Fill-up the details to create a new Pet record.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-rest-api/cp-mobile-dev-kit-rest-api.md:704]: BE:
```suggestion
3.  Fill in the details to create a new Pet record.
```
AE:
```suggestion
3.  Fill out the details to create a new Pet record.
```

**Comment by jitendrakansal** [comment]
@MichaelCzcz i have updated the tutorial with the changes you suggested, please check and do the needful.

**Comment by jitendrakansal** [comment]
@MichaelCzcz @thecodester  could you merge the changes if it looks fine?

**Pull request merged**